pkg/cmd/workspace/start.go

```diff
@@ -99,6 +110,7 @@ var StartCmd = &cobra.Command{
 func init() {
 	StartCmd.PersistentFlags().StringVarP(&startProjectFlag, "project", "p", "", "Start a single project in the workspace (project name)")
 	StartCmd.PersistentFlags().BoolVarP(&allFlag, "all", "a", false, "Start all workspaces")
+	StartCmd.PersistentFlags().BoolVarP(&ideCodeFlag, "code", "c", false, "Opens the workspace in the preferred IDE")
```

Comment:
Please sync the description from that in `create.go`

Reply:
done